examples/fuzzing/rt_n12/fuzz.py

```diff
@@ -0,0 +1,173 @@
+from qiling import *
+from qiling.const import *
+from qiling.extensions import afl
+from typing import Optional
+from qiling.os.const import *
+from qiling.extensions.coverage import utils as cov_utils
+import argparse
+
+payload_location = 0 # location in snapshot to be modified
+
+def my_send(ql:Qiling):
+    """
+    Hook to avoid writing to now no-longer connected socket
+    used to create this snapshot
+    """
+    params = ql.os.resolve_fcall_params({'sockfd': INT,'buf':POINTER, 'len':SIZE_T, 'flags':INT })
+    ql.log.info(buf)
```

Comment:
`buf` is not defined.
Did you mean `params[buf]`?

Reply:
> Did you mean params[buf]?

Yes

> Also consider a meaningful debug message and not just emitting the value out.

Done

Reply:
I had a typo; that should have been `params['buf']`, of course.

Reply:
Fixed.